tools-image/enki/pkg/action/converter.go

```diff
@@ -0,0 +1,13 @@
+package action
```

Comment:
These 2 files are not yet used. I will remove them an I will introduce them as part of this: https://github.com/kairos-io/kairos/issues/1721